app/components/Channels/ChannelCreateForm.js

```diff
@@ -114,152 +191,180 @@ class ChannelCreateForm extends React.Component {
     this.formApi = formApi
   }
 
+  renderFormFields() {
+    const { activeWalletSettings, isQueryingFees, searchQuery } = this.props
+
+    const formState = this.formApi.getState()
+    const speed = formState.values.speed || defaultSpeed
+    const fee = this.getFeeRate(speed)
+
+    return (
+      <Box>
+        <Box mb={3}>
+          <Text textAlign="justify">
+            <FormattedMessage {...messages.open_channel_form_description} />
+          </Text>
+        </Box>
+
+        <NodePubkeyInput
+          field="nodePubkey"
+          initialValue={searchQuery}
+          required
+          validateOnChange
+          validateOnBlur
+          disabled={Boolean(searchQuery)}
+        />
+
+        <Bar my={3} opacity={0.3} />
+
+        <CurrencyFieldGroup formApi={this.formApi} required />
+
+        <Bar my={3} opacity={0.3} />
+
+        <Flex justifyContent="space-between">
+          <Box>
+            <RadioGroup
+              field="speed"
+              label="Fee"
+              description={
+                <FormattedMessage
+                  {...messages[formState.values.speed.toLowerCase() + '_description']}
+                />
+              }
+              required
+            >
+              <Flex>
+                {speeds.map(speed => (
+                  <Radio
+                    key={speed}
+                    value={speed}
+                    label={<FormattedMessage {...messages[speed.toLowerCase()]} />}
+                    mr={4}
+                    mb={0}
+                  />
+                ))}
+              </Flex>
+            </RadioGroup>
+          </Box>
+          <Box>
+            {isQueryingFees ? (
+              <Flex ml="auto" alignItems="center" justifyContent="flex-end">
+                <Text mr={2}>
+                  <FormattedMessage {...messages.calculating} />
+                  &hellip;
+                </Text>
+                <Spinner color="lightningOrange" />
+              </Flex>
+            ) : !fee ? (
```

Comment:
Is there a way to avoid nested ternary ? Its generally not a good practice and is even prohibited in some lint rule bundles

Reply:
Updated.